app/policies/discussion_policy.rb

```diff
@@ -11,9 +11,9 @@ def show?
 
   def create?
     if Array.wrap(record).compact.any? { |d| d.board.section == 'zooniverse' }
-      writable? && of_posting_age?
+      writable? && confirmed? && of_posting_age?
```

Comment:
Do we still want the of_posting_age? restriction now that we have confirmed?

Reply:
Leaving this here because we can turn it off via deploy instead of removing the code.